CharacterCreation/01_04_Species_Opaleites.txt

```diff
@@ -155,21 +160,40 @@ Condrich does wear armor, they get an Armor Strength bonus of 20 to whichever
 armor that they wear.
     
 ===== Sea Devils =====
+
     
 Size: Medium
 Home System: Trioch IV
 
 Stats: +2 DEX
 
 Appearance:
+    Similar in size to the Mendars, the Sea Devils, or Samek in the Opaleite
+tongue, are named for their inherenet ability to bioluminesce. 
 
 == Available Traits ==
 
-Bioluminesce
-
-   [
+Light Dance:
+  
+    Sea Devils can harness their ability to bioluminesce to distract and 
+confuse enemies. Used in conjunction with darkness and quick movement, an
+experienced individual can alternate lighting scales from tip to tail to 
+misdirect an enemy.
+
+   Under low-light or darker, may use this ability while evading enemy
+fire to give you a +1 to hit. Enemies may roll Awareness vs 60 to see
+throught the deception. May also be used to attempt to distract an
+individual away from your teammates; enemy awareness vs. Acro[DEX] or Alth[DEX].
    
-Deep Glow
+Deep Glow:
 
-   [Light a 10m area around you with a dim light. +1 to enemies
-   trying to hit you]
+    Even those with Low-light vision cannot see in total darkness without
+aid. A Sea Devil may concentrate on their bioluminescent scales, brightening
+them to a leve that will allow others to see better in the darkness.
+    
+    The area 5m around you is increased two light levels, an additional
```

Comment:
"The light level of the area 5m around you is increased by two, and an..."

Reply:
Also, what are light levels? This was covered in @dfuchs2 's PR #800